@gregw the XML validation caught a bad XML issue in jetty-maven-plugin :+1:

I'll commit a fix to this branch for it.

```
[jetty-maven-plugin][jetty-10.0.x-4656-XmlConfiguration-Element-or-Attribute*]$ cat target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-start.out 
WARN  : System properties and/or JVM args set.  Consider using --dry-run or --exec
2020-03-12 10:18:43.584:INFO::main: Logging initialized @496ms to org.eclipse.jetty.util.log.StdErrLog
2020-03-12 10:18:43.863:INFO:oejs.Server:main: jetty-10.0.0-SNAPSHOT; built: 2020-03-12T13:59:50.484Z; git: df20446fdada659e1f3fbd2266cbe08c1f9f5349; jvm 11.0.6+10
2020-03-12 10:18:43.898:INFO:oejdp.ScanningAppProvider:main: Deployment monitor [file:///home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-base/webapps/] at interval 1
2020-03-12 10:18:43.916:WARN:oejx.XmlParser:main: ERROR@null line:7 col:25 : org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
2020-03-12 10:18:43.917:WARN:oejd.DeploymentManager:main: Unable to reach node goal: started
org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:396)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.validateDTDattribute(XMLDTDValidator.java:1389)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.addDTDDefaultAttrsAndValidate(XMLDTDValidator.java:1295)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.handleStartElement(XMLDTDValidator.java:1846)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.emptyElement(XMLDTDValidator.java:749)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:351)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2710)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:324)
	at org.eclipse.jetty.xml.XmlParser.parse(XmlParser.java:244)
	at org.eclipse.jetty.xml.XmlConfiguration.<init>(XmlConfiguration.java:253)
	at org.eclipse.jetty.deploy.providers.WebAppProvider$1.<init>(WebAppProvider.java:289)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.createContextHandler(WebAppProvider.java:288)
	at org.eclipse.jetty.deploy.App.getContextHandler(App.java:99)
	at org.eclipse.jetty.deploy.bindings.StandardDeployer.processBinding(StandardDeployer.java:38)
	at org.eclipse.jetty.deploy.AppLifeCycle.runBindings(AppLifeCycle.java:188)
	at org.eclipse.jetty.deploy.DeploymentManager.requestAppGoal(DeploymentManager.java:518)
	at org.eclipse.jetty.deploy.DeploymentManager.addApp(DeploymentManager.java:154)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.fileAdded(ScanningAppProvider.java:179)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.fileAdded(WebAppProvider.java:455)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider$1.fileAdded(ScanningAppProvider.java:66)
	at org.eclipse.jetty.util.Scanner.reportAddition(Scanner.java:789)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:756)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:641)
	at org.eclipse.jetty.util.Scanner.doStart(Scanner.java:540)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.doStart(ScanningAppProvider.java:151)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.deploy.DeploymentManager.startAppProvider(DeploymentManager.java:605)
	at org.eclipse.jetty.deploy.DeploymentManager.doStart(DeploymentManager.java:249)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.server.Server.start(Server.java:452)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:94)
	at org.eclipse.jetty.server.Server.doStart(Server.java:397)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.xml.XmlConfiguration.lambda$main$1(XmlConfiguration.java:1844)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1786)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.jetty.start.Main.invokeMain(Main.java:218)
	at org.eclipse.jetty.start.Main.start(Main.java:483)
	at org.eclipse.jetty.start.Main.main(Main.java:77)
Server@3b0fe47a{STARTING}[10.0.0-SNAPSHOT,sto=5000] - STARTING
+= QueuedThreadPool[qtp667821226]@27ce24aa{STOPPED,10<=0<=200,i=10,r=-1,q=0}[NO_TRY] - STOPPED
|  +> threads size=0
+- org.eclipse.jetty.io.ArrayByteBufferPool@448c8166
+= ScheduledExecutorScheduler@6743e411{STOPPED} - STOPPED
+= HandlerCollection@4470fbd6{STOPPED} - STOPPED
|  += ContextHandlerCollection@15d49048{STOPPED} - STOPPED
|  += DefaultHandler@7098b907{STOPPED} - STOPPED
+- org.eclipse.jetty.maven.plugin.ServerListener@503f91c3
+= ServerConnector@13526e59{HTTP/1.1, (http/1.1)}{0.0.0.0:0} - STOPPED
|  +~ QueuedThreadPool[qtp667821226]@27ce24aa{STOPPED,10<=0<=200,i=10,r=-1,q=0}[NO_TRY] - STOPPED
|  +? ScheduledExecutorScheduler@6743e411{STOPPED} - STOPPED
|  +- org.eclipse.jetty.io.ArrayByteBufferPool@448c8166
|  +- org.eclipse.jetty.server.AbstractConnector$1@36804139
|  +? HttpConnectionFactory@1de76cc7[HTTP/1.1] - STOPPED
|  |  +- HttpConfiguration@2438dcd{32768/8192,8192/8192,https://:0,[]}
|  |     +> customizers size=0
|  |     +> formEncodedMethods size=2
|  |     |  +> POST
|  |     |  +> PUT
|  |     +> outputBufferSize=32768
|  |     +> outputAggregationSize=8192
|  |     +> requestHeaderSize=8192
|  |     +> responseHeaderSize=8192
|  |     +> headerCacheSize=1024
|  |     +> secureScheme=https
|  |     +> securePort=0
|  |     +> idleTimeout=-1
|  |     +> sendDateHeader=true
|  |     +> sendServerVersion=true
|  |     +> sendXPoweredBy=false
|  |     +> delayDispatchUntilContent=true
|  |     +> persistentConnectionsEnabled=true
|  |     +> maxErrorDispatches=10
|  |     +> minRequestDataRate=0
|  |     +> minResponseDataRate=0
|  |     +> requestCookieCompliance=org.eclipse.jetty.http.CookieCompliance@7975d1d8
|  |     +> responseCookieCompliance=org.eclipse.jetty.http.CookieCompliance@7975d1d8
|  |     +> notifyRemoteAsyncErrors=true
|  += SelectorManager@ServerConnector@13526e59{HTTP/1.1, (http/1.1)}{0.0.0.0:0} - STOPPED
|  +- org.eclipse.jetty.maven.plugin.ServerConnectorListener@24105dc5
+= DeploymentManager@5167f57d{FAILED} - FAILED
|  += class org.eclipse.jetty.deploy.providers.WebAppProvider@64b0598[file:///home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-base/webapps/] - STARTED
|     +- java.util.HashMap@c96c9bc6{size=1}
|     |  +@ /home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-base/webapps/maven.xml = App[null,/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-base/webapps/maven.xml]
|     +- class org.eclipse.jetty.deploy.PropertiesConfigurationManager@3e2055d6{null}
|     |  +> java.util.HashMap@0{size=0}
|     += Scanner@50029372{STARTED} - STARTED
+? ErrorHandler@e3b3b2f{STOPPED} - STOPPED
+> startJarLoader@67205a84
   +> URLs size=26
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-servlet-api-4.0.3.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-http-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-server-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-xml-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-util-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-io-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-jndi-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-security-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-servlet-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-webapp-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-plus-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jakarta.transaction-api-1.3.2.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-annotations-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/annotations/asm-7.2.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/annotations/asm-analysis-7.2.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/annotations/asm-commons-7.2.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/annotations/asm-tree-7.2.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/annotations/jakarta.annotation-api-1.3.4.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-base/lib/maven/plugin.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/apache-jsp/org.eclipse.jdt.ecj-3.19.0.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/apache-jsp/org.eclipse.jetty.apache-jsp-10.0.0-SNAPSHOT.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/apache-jsp/org.mortbay.jasper.apache-el-9.0.29.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/apache-jsp/org.mortbay.jasper.apache-jsp-9.0.29.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/apache-jstl/org.apache.taglibs.taglibs-standard-impl-1.2.5.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/apache-jstl/org.apache.taglibs.taglibs-standard-spec-1.2.5.jar
   |  +> file:/home/joakim/code/jetty/jetty.project-10.0.x/jetty-maven-plugin/target/it/jetty-start-distro-mojo-it/jetty-simple-webapp/target/jetty-home-10.0.0-SNAPSHOT/lib/jetty-deploy-10.0.0-SNAPSHOT.jar
   +> jdk.internal.loader.ClassLoaders$AppClassLoader@277050dc
      +> packages size=3
      |  +> package org.eclipse.jetty.start.config
      |  +> package org.eclipse.jetty.start.shaded.util
      |  +> package org.eclipse.jetty.start
      +> jdk.internal.loader.ClassLoaders$PlatformClassLoader@2f01783a
         +> packages size=2
            +> package sun.util.resources.provider
            +> package sun.util.resources.cldr.provider
key: +- bean, += managed, +~ unmanaged, +? auto, +: iterable, +] array, +@ map, +> undefined
2020-03-12 10:18:43.956:WARN:oejx.XmlConfiguration:main: 
java.security.PrivilegedActionException: org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1786)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.jetty.start.Main.invokeMain(Main.java:218)
	at org.eclipse.jetty.start.Main.start(Main.java:483)
	at org.eclipse.jetty.start.Main.main(Main.java:77)
Caused by: 
org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:396)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.validateDTDattribute(XMLDTDValidator.java:1389)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.addDTDDefaultAttrsAndValidate(XMLDTDValidator.java:1295)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.handleStartElement(XMLDTDValidator.java:1846)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.emptyElement(XMLDTDValidator.java:749)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:351)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2710)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:324)
	at org.eclipse.jetty.xml.XmlParser.parse(XmlParser.java:244)
	at org.eclipse.jetty.xml.XmlConfiguration.<init>(XmlConfiguration.java:253)
	at org.eclipse.jetty.deploy.providers.WebAppProvider$1.<init>(WebAppProvider.java:289)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.createContextHandler(WebAppProvider.java:288)
	at org.eclipse.jetty.deploy.App.getContextHandler(App.java:99)
	at org.eclipse.jetty.deploy.bindings.StandardDeployer.processBinding(StandardDeployer.java:38)
	at org.eclipse.jetty.deploy.AppLifeCycle.runBindings(AppLifeCycle.java:188)
	at org.eclipse.jetty.deploy.DeploymentManager.requestAppGoal(DeploymentManager.java:518)
	at org.eclipse.jetty.deploy.DeploymentManager.addApp(DeploymentManager.java:154)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.fileAdded(ScanningAppProvider.java:179)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.fileAdded(WebAppProvider.java:455)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider$1.fileAdded(ScanningAppProvider.java:66)
	at org.eclipse.jetty.util.Scanner.reportAddition(Scanner.java:789)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:756)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:641)
	at org.eclipse.jetty.util.Scanner.doStart(Scanner.java:540)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.doStart(ScanningAppProvider.java:151)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.deploy.DeploymentManager.startAppProvider(DeploymentManager.java:605)
	at org.eclipse.jetty.deploy.DeploymentManager.doStart(DeploymentManager.java:249)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.server.Server.start(Server.java:452)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:94)
	at org.eclipse.jetty.server.Server.doStart(Server.java:397)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.xml.XmlConfiguration.lambda$main$1(XmlConfiguration.java:1844)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1786)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.jetty.start.Main.invokeMain(Main.java:218)
	at org.eclipse.jetty.start.Main.start(Main.java:483)
	at org.eclipse.jetty.start.Main.main(Main.java:77)
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.jetty.start.Main.invokeMain(Main.java:218)
	at org.eclipse.jetty.start.Main.start(Main.java:483)
	at org.eclipse.jetty.start.Main.main(Main.java:77)
Caused by: java.security.PrivilegedActionException: org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1786)
	... 7 more
Caused by: org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:396)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.validateDTDattribute(XMLDTDValidator.java:1389)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.addDTDDefaultAttrsAndValidate(XMLDTDValidator.java:1295)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.handleStartElement(XMLDTDValidator.java:1846)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.emptyElement(XMLDTDValidator.java:749)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:351)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2710)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:324)
	at org.eclipse.jetty.xml.XmlParser.parse(XmlParser.java:244)
	at org.eclipse.jetty.xml.XmlConfiguration.<init>(XmlConfiguration.java:253)
	at org.eclipse.jetty.deploy.providers.WebAppProvider$1.<init>(WebAppProvider.java:289)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.createContextHandler(WebAppProvider.java:288)
	at org.eclipse.jetty.deploy.App.getContextHandler(App.java:99)
	at org.eclipse.jetty.deploy.bindings.StandardDeployer.processBinding(StandardDeployer.java:38)
	at org.eclipse.jetty.deploy.AppLifeCycle.runBindings(AppLifeCycle.java:188)
	at org.eclipse.jetty.deploy.DeploymentManager.requestAppGoal(DeploymentManager.java:518)
	at org.eclipse.jetty.deploy.DeploymentManager.addApp(DeploymentManager.java:154)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.fileAdded(ScanningAppProvider.java:179)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.fileAdded(WebAppProvider.java:455)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider$1.fileAdded(ScanningAppProvider.java:66)
	at org.eclipse.jetty.util.Scanner.reportAddition(Scanner.java:789)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:756)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:641)
	at org.eclipse.jetty.util.Scanner.doStart(Scanner.java:540)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.doStart(ScanningAppProvider.java:151)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.deploy.DeploymentManager.startAppProvider(DeploymentManager.java:605)
	at org.eclipse.jetty.deploy.DeploymentManager.doStart(DeploymentManager.java:249)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.server.Server.start(Server.java:452)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:94)
	at org.eclipse.jetty.server.Server.doStart(Server.java:397)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.xml.XmlConfiguration.lambda$main$1(XmlConfiguration.java:1844)
	... 9 more
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.jetty.start.Main.invokeMain(Main.java:218)
	at org.eclipse.jetty.start.Main.start(Main.java:483)
	at org.eclipse.jetty.start.Main.main(Main.java:77)
Caused by: java.security.PrivilegedActionException: org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1786)
	... 7 more
Caused by: org.xml.sax.SAXParseException; lineNumber: 7; columnNumber: 25; Attribute value "wac" of type ID must be unique within the document.
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:135)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:396)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.validateDTDattribute(XMLDTDValidator.java:1389)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.addDTDDefaultAttrsAndValidate(XMLDTDValidator.java:1295)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.handleStartElement(XMLDTDValidator.java:1846)
	at java.xml/com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.emptyElement(XMLDTDValidator.java:749)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:351)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2710)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:324)
	at org.eclipse.jetty.xml.XmlParser.parse(XmlParser.java:244)
	at org.eclipse.jetty.xml.XmlConfiguration.<init>(XmlConfiguration.java:253)
	at org.eclipse.jetty.deploy.providers.WebAppProvider$1.<init>(WebAppProvider.java:289)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.createContextHandler(WebAppProvider.java:288)
	at org.eclipse.jetty.deploy.App.getContextHandler(App.java:99)
	at org.eclipse.jetty.deploy.bindings.StandardDeployer.processBinding(StandardDeployer.java:38)
	at org.eclipse.jetty.deploy.AppLifeCycle.runBindings(AppLifeCycle.java:188)
	at org.eclipse.jetty.deploy.DeploymentManager.requestAppGoal(DeploymentManager.java:518)
	at org.eclipse.jetty.deploy.DeploymentManager.addApp(DeploymentManager.java:154)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.fileAdded(ScanningAppProvider.java:179)
	at org.eclipse.jetty.deploy.providers.WebAppProvider.fileAdded(WebAppProvider.java:455)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider$1.fileAdded(ScanningAppProvider.java:66)
	at org.eclipse.jetty.util.Scanner.reportAddition(Scanner.java:789)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:756)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:641)
	at org.eclipse.jetty.util.Scanner.doStart(Scanner.java:540)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.deploy.providers.ScanningAppProvider.doStart(ScanningAppProvider.java:151)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.deploy.DeploymentManager.startAppProvider(DeploymentManager.java:605)
	at org.eclipse.jetty.deploy.DeploymentManager.doStart(DeploymentManager.java:249)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:174)
	at org.eclipse.jetty.server.Server.start(Server.java:452)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:124)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:94)
	at org.eclipse.jetty.server.Server.doStart(Server.java:397)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:96)
	at org.eclipse.jetty.xml.XmlConfiguration.lambda$main$1(XmlConfiguration.java:1844)
	... 9 more

Usage: java -jar $JETTY_HOME/start.jar [options] [properties] [configs]
       java -jar $JETTY_HOME/start.jar --help  # for more information
```

--- a/jetty-maven-plugin/src/main/resources/maven.xml
+++ b/jetty-maven-plugin/src/main/resources/maven.xml
@@ -4,9 +4,13 @@
 <Configure id="wac" class="org.eclipse.jetty.maven.plugin.MavenWebAppContext">
 
   <Call class="org.eclipse.jetty.maven.plugin.WebAppPropertyConverter" name="fromProperties">
-    <Arg><Ref id="wac"/></Arg>
+    <Arg>
+      <Ref refid="wac" />
+    </Arg>
     <Arg><Property name="jetty.base" default="."/>/etc/maven.props</Arg>
-    <Arg><Ref id="Server"/></Arg>
+    <Arg>
+      <Ref refid="Server" />
+    </Arg>
     <Arg></Arg>
   </Call>
 

